lambdas/service/app.py

```diff
@@ -85,7 +85,95 @@
 
 openapi_spec = {
     'info': {
-        'description': 'This should probably be really long',
+        'description': format_description('''
+
+            # Overview
+
+            Azul is a REST web service for querying metadata associated with
+            both experimental and analysis data stored in the [HCA Data Store
+            (DSS)](https://github.com/HumanCellAtlas/data-store). In order to
+            deliver response times that make it suitable for interactive use
+            cases, it limits the set of metadata properties that it exposes
+            for sorting, filtering and aggregation. It provides a uniform view
+            of the metadata over a range of schemas, effectively shielding
+            clients from schema changes, but does so at the expense of detail.
+
+            Azul denormalizes and aggregates metadata into several different
+            indices for selected entity types:
+
+             - [projects](#operations-Repository-get_repository_projects)
+
+             - [samples](#operations-Repository-get_repository_samples)
+
+             - [files](#operations-Repository-get_repository_files)
+
+             - [bundles](#operations-Repository-get_repository_bundles)
+
+            Azul provides the ability to download metadata in tabular form via
+            the [Manifests](#/Manifests) endpoints. The resulting manifests
+            include links to associated data files and can be used by the
+            [DCP CLI](https://github.com/HumanCellAtlas/dcp-cli) to download the
+            listed files. Manifests can be generated for a selection of files
+            using filters. These filters are interchangeable with the filters
+            used by the [Repository](#/Repository) endpoints.
+
+            Azul also provides a
+            [summary](#operations-Repository-get_repository_summary) view of
+            indexed data.
+
+            ## Data model
+
+            Any index, when queried, returns a JSON array of hits. Each hit
+            represents a metadata entity. Nested in each hit is a summary of the
+            properties of entities associated with the hit. An entity is
+            associated either by a direct edge in the original metadata graph,
+            or indirectly as a series of edges. The nested properties are
+            grouped by the type of the associated entity. The properties of all
+            data files associated with a particular sample, for example, are
+            listed under `hits[*].files` in a `/repository/samples` response. It
+            is important to note that while each _hit_ represents a discrete
+            entity, the properties nested within that hit are the result of an
+            aggregation over potentially many associated entities.
+
+            To illustrate this, suppose you had a data file that is part of
```

Comment:
"you had" might be awkward

Reply:
Agreed. I replaced this with "consider".